### PR TITLE
GNN fraud detection notebook fix

### LINKED ISSUE #1265: [BUG]: Missing `map_node_id` in GNN Fraud Detection training notebook
### Version

23.11

### Which installation method(s) does this occur on?

Docker

### Describe the bug.

An `ImportError`  occurs in the first cell of the GNN Fraud Detection training notebook because `map_node_id` does not exist in `training.py`.

### Minimum reproducible example

1. `cd` to `models/training-tuning-scripts/fraud-detection-models`

2. Create or update conda environment with required dependencies:

Create new environment:
```shell
conda env create -f ./requirements.yml -n fraud-detection
conda activate fraud-detection
```

Update existing environment:
```shell
conda env update -f ./requirements.yml
```

3. Start JupyterLab

4. Open and run `gnn-fraud-detection-training.ipynb`

### Relevant log output

<details><summary>Click here to see error details</summary><pre>

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 23
     21 from tqdm import trange
     22 from xgboost import XGBClassifier
---> 23 from training import (get_metrics, evaluate, init_loaders, build_fsi_graph,
     24                      map_node_id, prepare_data, save_model, train)

ImportError: cannot import name 'map_node_id' from 'training' (/workspace/models/training-tuning-scripts/fraud-detection-models/training.py)

</pre></details>

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by efajardo-nv**
## Description
- Add notebook fixes from @tzemicheal
- Migrate remaining pandas code to cudf

Closes #1265 

## By Submitting this PR I confirm:
- I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- When the PR is ready for review, new or existing tests cover these changes.
- When the PR is ready for review, the documentation is up to date with these changes.

### CODE REVIEW

**Review by tzemicheal — APPROVED** [review]
The fix is LGTM!

File: models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb [hunk 1]
```diff
@@ -50,16 +50,16 @@
    "source": [
```
Comment (mdemoret-nv) [inline models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb:50]: <p>Make sure these are sorted the same way they would be by <code>isort</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nv-morpheus/Morpheus/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (mdemoret-nv) [inline models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb:50]: <p>Should we just remove this block?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nv-morpheus/Morpheus/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (mdemoret-nv) [inline models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb:50]: <p>Lets either remove this block or uncomment it out. Users can optionally run inference if they want. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nv-morpheus/Morpheus/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (efajardo-nv) [reply models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb:50]: removed

Comment (efajardo-nv) [reply models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb:50]: uncommented

**Pull request merged**